typescript/infra/scripts/verify-validator.ts

```diff
@@ -301,4 +375,44 @@ function streamToString(stream: NodeJS.ReadableStream): Promise<string> {
   });
 }
 
+////
+// Bootstrapper
+////
+function getArgs() {
+  return yargs(process.argv.slice(2))
+    .alias('a', 'address')
+    .describe('a', 'address of the validator to inspect')
+    .demandOption('a')
+    .string('a')
+    .alias('p', 'prospective')
+    .describe('p', 'S3 bucket of the prospective validator')
+    .demandOption('p')
+    .string('p')
+    .alias('c', 'control')
+    .describe('c', 'S3 bucket of the the known (control) validator')
+    .demandOption('c')
+    .string('c').argv;
+}
+
+async function main() {
+  const {
+    a: validatorAddress,
+    p: prospectiveBucket,
+    c: controlBucket,
+  } = await getArgs();
+
+  const validator = new Validator(
+    validatorAddress,
+    prospectiveBucket,
+    controlBucket,
+  );
+
+  try {
+    await validator.validate();
+  } catch (err) {
+    console.error(err);
+    Process.exit(1);
```

Comment:
Is this capital P Process expected here?

Reply:
When I test in the node shell `process` is the valid one and `Process` is `undefined`.